tensorflow/python/ops/signal/mel_ops.py

```diff
@@ -102,6 +102,9 @@ def linear_to_mel_weight_matrix(num_mel_bins=20,
   `num_spectrogram_bins` linearly sampled frequency information from
   `[0, sample_rate / 2]` into `num_mel_bins` frequency information from
   `[lower_edge_hertz, upper_edge_hertz]` on the [mel scale][mel].
+  
+  The implementation follows a mel scale formula, 2595 * log10(1 + f/700), where
```

Comment:
how about using TeX to typeset the formula? 
I'd also suggest leading with the HTK bit? Something like:

```
This function follows the [Hidden Markov Model Toolkit (HTK)](http://htk.eng.cam.ac.uk/) convention, defining the mel scale in terms of a frequency in hertz according to the following formula:

$$\textrm{mel}(f) = 2595 * \textrm{log}_{10}(1 + \frac{f}{700})$$
```

Reply:
Will do something like that in another PR. Thanks :)